### ISSUE TITLE
[Bug]: Logs are not properly emitted to the console.

### ISSUE THREAD

**Issue by shivamkj** [taskforcesh/bullmq#2605]
### Version

5.7.15

### Platform

NodeJS

### What happened?

Console.log doesn't produces output to console properly. Only for some events it shows the logs, others events even though gets processed successfully but no logs are emitted. For example, I add 4 same events to the queue, all four gets processed that I confirmed from `@bull-board/api`. But logs were only emitted to console for the first and 4th event. 

### How to reproduce.

Below is a sample code that reproduces this error on my machine.

```
import { Queue, Worker } from 'bullmq'
import fastify from 'fastify'

const queueName = 'qnifyQueue'
const tasks = { cronJob: 'cronJob' }
const connection = {
  host: process.env.REDIS_HOST,
  port: process.env.REDIS_PORT,
}

export const asyncQueue = new Queue(queueName, { connection: connection })

export const worker = new Worker(
  queueName,
  async ({ name, data }) => {
    console.log('processing', name, data)
    await processJob()
  },
  { connection }
)

worker.on('failed', (job, err) => console.error(`Job with Name:${job.name}, Id:${job.id} has failed with err`, err))
worker.on('completed', (job) => console.log(`Job with Name:${job.name}, Id:${job.id} has completed`))

const processJob = () => console.log('running cron job')

export async function addJob(request, reply) {
  await asyncQueue.add(tasks.cronJob, request.body)
  reply.send({ success: true })
}

const fastifyInstance = fastify()
fastifyInstance.get('/process', addJob)
fastifyInstance.listen({ port: 3003 }, (err, _) => {
  if (err) {
    console.error(err)
    process.exit(1)
  }
  console.log('Server started')
})
```

### Relevant log output

```shell
Server started
processing cronJob {}
running cron job
Job with Name:cronJob, Id:1 has completed
processing cronJob {}
running cron job
Job with Name:cronJob, Id:4 has completed
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by manast** [comment]
Most likely you have another worker consuming jobs and you are unaware of it. If you cannot figure it out, please strip down the test case without Fastify and any other dependencies and we will look into it.

**Comment by shivamkj** [comment]
Strange but it looks VS Code is the culprit, I flushed all Redis keys and then restarted the system, but as soon as I open VSCode, 
BullMQ keys are created in Redis, without me running anything, VSCode probably executes the code which creates the first worker without my knowledge. Now when I execute the code & start server, 1 worker gets created and so 2 workers are working and causing the issue. Working fine on server. Closing as it's not an issue with BullMQ but with VSCode.
